account_financial_report_qweb/report/general_ledger.py

```diff
@@ -280,14 +280,9 @@ def _get_account_sub_subquery_sum_amounts(
                 SUM(ml.debit) AS debit,
                 SUM(ml.credit) AS credit,
                 SUM(ml.balance) AS balance,
-                CASE
-                    WHEN COUNT(c.name) = COUNT(1)
-                        AND MIN(c.name) = MAX(c.name)
```

Comment:
Here and in the following occurrencies, the code `AND MIN(c.name) = MAX(c.name)` was used to be sure that there is only one currency in all the grouped move lines. 
Without it, it would incorrectly sum the amounts of different currencies

Reply:
Well, and that is the point. You are not supposed to sum totals in move lines when the associated account is not maintained using a secondary currency. You just show the individual amounts in different currency.